### ISSUE TITLE
[Feature] Include markdown tables

### ISSUE THREAD

**Issue by tysino** [TechbeeAT/jtxBoard#146]
**Is your feature request related to a problem? Please describe.**
Weirdly, table syntax is recognized, but the formatting characters (`|`, `-`) are just removed in the rendered markdown.

**Describe the solution you'd like**
Implement github-style rendered tables (maybe just a library change is necessary).

**Describe alternatives you've considered**
Other option: show the formatting characters in the rendered markdown (which is ugly, but at least not unexpected).

**Additional context**
[The Android client for Nextcloud notes](https://github.com/stefan-niedermann/nextcloud-notes) supports this, if you are looking for a library.

PS: Thank you for the awesome app.

**Comment by patrickunterwegs** [comment]
Thanks @tysino for the feedback 🙂 

@ArnyminerZ , would you maybe have the time to check this out after your exam period? Could we move this to the MarkdownText project? 🙂

**Comment by ArnyminerZ** [comment]
The issue here is that we are working with `AnnotatedString`, I don't see how we could create a table only with characters. Maybe with some ASCII magic... But we should investigate a little bit. The other way could be generating an image and rendering it... I'll take a look at the options. We should move the issue to MarkdownText then.
The good thing is that the table is already being parsed by the library, so it's just a rendering issue.

**Comment by patrickunterwegs** [comment]
Yes, I also wouldn't go too far with the markdown support, but maybe we can also just keep the tags for now!

**Comment by ArnyminerZ** [comment]
I have now released MarkdownText 1.2.3, it fixes temporally the issue by rendering the table as raw Markdown. Once I finish exams I will work on a proper implementation of tables.

**Comment by patrickunterwegs** [comment]
Thanks @ArnyminerZ  🙂
